README.md

```diff
@@ -69,6 +75,8 @@ After writing this method, our new result should be:
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
 
+The strip_punctuation method should be called before the loop, because we want to remove punctuation before we add to the values of happy or sad variables. It could technically be put anywhere before the loop though.
+
```

Comment:
True. This is a great observation. I should have additionally asked, "Should you call strip_punctuation inside or outside of analyze_mood? Why?"